Shared/Supporting Files/Views/CategoryView.swift

```diff
@@ -37,7 +37,7 @@ struct CategoryView: View {
             if !isSearching {
                 CategoryGridView(samples: samples)
             } else {
-                SampleListView(samples: displayedSamples)
+                SampleListView(samples: displayedSamples, shouldShowCategory: true)
```

Comment:
<img width="213" alt="Screenshot 2023-07-27 at 10 39 14" src="https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/16397058/edaf5b6f-ad92-4cf3-b69b-e23496be7b06">

This is again more of a design note and not technical so no need to hold the PR up on this but I'm not sure I feel that showing the category here is beneficial. Is the point that displaying the category would help a user find the specific sample faster in the future by knowing the correct category?

Reply:
The idea behind showing the category in the search was to make the samples more discoverable. If a user wanted to know what category a sample was in so that they could access it through the category grid, they would just have to search the name to find the category instead of having to hunt through all the categories until they found it. This would allow them to find samples that are similar to a sample they found through the search without looking through all the samples to find the correct category.